bots/Bots.md

```diff
@@ -405,6 +409,58 @@ arn:aws:s3:::my-bucket/my-logs/
 log delivery policy name is set as: vpcFlowLogDelivery log delivery role
 is set as: vpcFlowLogDelivery
 
+## s3\_block\_public\_access
+
+What it does: turn on S3 Bucket Block public access
+
+Usage:  s3_block_public_access <BlockNewPublicAcls> <BlockNewPublicPolicy>
+
+Example: AUTO: s3_block_public_access <true|false|-> <true|false|->
+When - set for ignore the specific attribute if its optional so usage will be
+
+Limitations: none
+
+Notes:
+    -  before running this bot, ensure that your applications will work correctly without public access
+    
+Another Examples:
```

Comment:
Please ask Guy to go over those examples

Reply:
Guy instructed me through the bot changes and sent the bot to the client himself after approving it, and the usage ofcourse.

Reply:
fixed